src/branch-name/branch-name.scss

```diff
@@ -8,6 +8,7 @@
   padding: 2px 6px;
   font: 12px $mono-font;
   color: var(--color-fg-muted);
+  word-break: break-all;
```

Comment:
@simurai I did a little [reading](https://developer.mozilla.org/en-US/docs/Web/CSS/word-break) on `word-break` (did you know `break-word` is deprecated? 😮) and I'm wondering if this would solve for the `-` dash scenario you mentioned in the PR description:

```suggestion
 overflow-wrap: anywhere;
 word-break: normal;
```

Reply:
@langermank `overflow-wrap: anywhere;` still seems [not supported](https://developer.mozilla.org/en-US/docs/Web/CSS/overflow-wrap#browser_compatibility) in Safari.

Another thing with dashes/hyphens is that if the branch name starts with a dash/hyphen and follows with dots, underlines, slashes... it prioritizes that first dash/hyphen and leaving a big gap. So maybe for this specific case with branch names, using `break-all` and just fill the line as much as possible might still be the better option?

`overflow-wrap: anywhere;` | `word-break: break-all;`
--- | ---
![Screen Shot 2022-03-30 at 13 19 23](https://user-images.githubusercontent.com/378023/160751307-ea157d85-cf6a-43d1-9ff8-dc5256e0c0f4.png) | ![Screen Shot 2022-03-30 at 13 20 05](https://user-images.githubusercontent.com/378023/160751315-37238b91-2271-42b0-ad56-53653301a52e.png)

Reply:
Oh shoot! I didn't check the browser support. I would just keep `break-all` then. It's not deprecated, `break-word` is (sorry, didn't mean to confuse you). I was just surprised by that as I was thinking of recommending it!